src/zrc/api/tests/test_zaakinformatieobjecten.py

```diff
@@ -1,50 +1,252 @@
+        })
+
+        expected_response = content.copy()
+        expected_response.update({
+            'url': f'http://testserver{expected_url}',
+            'titel': titel,
+            'beschrijving': beschrijving,
+            'registratiedatum': '2018-09-19T10:25:19Z',
+            'aardRelatieWeergave': RelatieAarden.labels[RelatieAarden.hoort_bij],
+        })
+        self.assertEqual(response.json(), expected_response)
+
+    @freeze_time('2018-09-20 12:00:00')
+    def test_registratiedatum_ignored(self):
+        zaak = ZaakFactory.create()
+        zaak_url = reverse('zaak-detail', kwargs={
+            'version': '1',
+            'uuid': zaak.uuid,
+        })
+
+        content = {
+            'informatieobject': INFORMATIEOBJECT,
+            'zaak': 'http://testserver' + zaak_url,
+            'registratiedatum': '2018-09-19T12:25:20+0200',
+        }
+
+        # Send to the API
+        self.client.post(self.list_url, content)
+
+        oio = ZaakInformatieObject.objects.get()
+
+        self.assertEqual(
+            oio.registratiedatum,
+            datetime(2018, 9, 20, 12, 0, 0).replace(tzinfo=timezone.utc)
+        )
+
+    def test_duplicate_object(self):
+        """
+        Test the (informatieobject, object) unique together validation.
+        """
+        zio = ZaakInformatieObjectFactory.create(
+            informatieobject=INFORMATIEOBJECT
+        )
+        zaak_url = reverse('zaak-detail', kwargs={
+            'version': '1',
+            'uuid': zio.zaak.uuid,
+        })
+
+        content = {
+            'informatieobject': zio.informatieobject,
+            'zaak': f'http://testserver{zaak_url}',
+        }
+
+        # Send to the API
+        response = self.client.post(self.list_url, content)
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST, response.data)
+        error = get_validation_errors(response, 'nonFieldErrors')
+        self.assertEqual(error['code'], 'unique')
+
+    def test_read_zaak(self):
+        zio = ZaakInformatieObjectFactory.create(
+            informatieobject=INFORMATIEOBJECT
+        )
+        # Retrieve from the API
+
+        zio_detail_url = reverse('zaakinformatieobject-detail', kwargs={
+            'version': '1',
+            'uuid': zio.uuid,
+        })
+        response = self.client.get(zio_detail_url)
+
+        # Test response
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        zaak_url = reverse('zaak-detail', kwargs={
+            'version': '1',
+            'uuid': zio.zaak.uuid,
+        })
+
+        expected = {
+            'url': f'http://testserver{zio_detail_url}',
+            'informatieobject': zio.informatieobject,
+            'zaak': f'http://testserver{zaak_url}',
+            'aardRelatieWeergave': RelatieAarden.labels[RelatieAarden.hoort_bij],
+            'titel': '',
+            'beschrijving': '',
+            'registratiedatum': dt_to_api(zio.registratiedatum),
```

Comment:
use `@freeze_time` on the test and make this an explicit assertion instead - keeps the tests a lot more readable and easier to reason about